tests/acceptance/features/apiActivities/activitiesByFileId.feature

```diff
@@ -472,3 +472,809 @@ Feature: check activities
         }
       }
       """
+
+
+  Scenario: check activities of destination folder after moving a file into it using file-id
```

Comment:
This scenario can be merged with the one above
https://github.com/owncloud/ocis/blob/3847f46dedd7c5e88db6da335ee7a8b4dcfc38f4/tests/acceptance/features/apiActivities/activitiesByFileId.feature#L364